### ISSUE TITLE
Proposal: move all stock entry params into the "menu" of stock entry

### ISSUE THREAD

**Issue by jniles** [Third-Culture-Software/bhima#6638]
I think we would have a much better user experience if we move all the stock entry parameters into the stock entry menu.  Here is why:

1. Most users will never touch these parameters.  They take up valuable real estate that could be used for what the user is actually interacting with - creating lots in stock.
2. User fatigue.  I don't think anyone will pay any attention to these because there is a lot of options here.  They pop up and pop down.
3. Many of these apply across stock entry session.  If you are a big depot, you may always want the "global quantity" to show up.  If you are small pharmacy, like the pharmacy d'usag you just want to import the stock sent to you and you will never use it.

For reference, this is what I am talking about:
![image](https://user-images.githubusercontent.com/896472/170468076-ba5394c2-8303-4e15-a1d3-64563eb10d2a.png)

Therefore, I propose we move these options into a "Stock Entry Settings" page that is in the menu that can better explain what each of these settings will do.

**Comment by jniles** [comment]
@jmcameron what do you think of this proposal?

**Comment by jmcameron** [comment]
Sounds good to move the settings -- but not the global quantity or unit price since these are lot-specific.

**Comment by jniles** [comment]
What do you think about putting an option to allow modification to the unit prices and global quantity in the menu, and lock those by default?

**Comment by jmcameron** [comment]
That might work for the unit price.  BTW, should that option have a permission/role?  Should anyone (eg pharmacist) be able to change the unit price?

Regarding the global quantity:  We should handle global quantity better than we do.   
 - If a stock entry is from a purchase order, requistion or shipment, then perhaps the global quantity should be locked and do not let the user change it.  
 - However, for entry from donations, integrations, etc (eg, there is no prior info on the available quantity), then we should probably hide and ignore the global quantity.
 - Should we rename "global quantity" to "available quantity"?

**Comment by jniles** [comment]
Yeah, you are definitely right - that should be a user/role/permission option to enable price edits.

I agree on all points related to the "Global Quantity".
